libbeat/docs/communitydashboards.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[community-beat-dashboards]]
+== Community Beat dashboards
+
+The open source community has been hard at work developing new Beats. You can check
```

Comment:
I agree with you @dedemorton, and I suggest for now to recommend users to create a topic on discuss when they want to share their own Beat dashboards.